src/nidcpower/metadata/enums.py

```diff
@@ -495,6 +574,24 @@
             }
         ]
     },
+    'TBoolean': {
```

Comment:
Where is this enum type used?

Reply:
This was added by mistake, I will remove it.